docs/1.guide/2.config.md

```diff
@@ -31,16 +32,23 @@ Working directory where paths are resolved from.
 > [!TIP]
 > You can use `--dir` to override with CLI.
 
-### `file`
+### `files`
 
-- Type: `string`
+- Type: `string|string[]`
 - Default: `README.md`
 
 The markdown file name or path (relative to `dir`).
 
 > [!TIP]
 > You can use `--file` to override with CLI.
 
+### `dirs`
+
+- Type: `string[]`
+- Default: `[]`
+
+Directory paths to scan for markdown files (relative to `dir`).
```

Comment:
```suggestion
Directory paths to scan for markdown files.
```

Reply:
I want the user to know it's relative to the dir they set. I don't want them to set a full path so it won't be confusing